packages/app/app/components/FavoriteTracksView/FavoriteTracksView.test.tsx

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
Can you put those tests here instead? https://github.com/nukeop/nuclear/blob/master/packages/app/app/containers/FavoritesContainer/FavoritesContainer.tracks.test.tsx

Testing the container will give us more reliable coverage than testing the presentational component